README.md

```diff
@@ -121,13 +123,13 @@ Once you're done with the above setup (no matter which method you used), run `np
     Checking theme for possible violations:
```

Comment:
I'll create the composer scripts that one can run to check the phpcs, once I do this we'll update this part. Aliasing is one extra step, and composer is already there :)

Reply:
Great! 👍